frontend/src/app/components/wp-table/timeline/container/wp-timeline-container.directive.ts

```diff
@@ -57,9 +57,15 @@ import {debugLog, timeOutput} from "core-app/helpers/debug_output";
 import {RenderedWorkPackage} from "core-app/modules/work_packages/render-info/rendered-work-package.type";
 import {HalEventsService} from "core-app/modules/hal/services/hal-events.service";
 import {WorkPackageNotificationService} from "core-app/modules/work_packages/notifications/work-package-notification.service";
-import {combineLatest} from "rxjs";
+import {combineLatest, merge} from "rxjs";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 import {WorkPackagesTableComponent} from "core-components/wp-table/wp-table.component";
+import {GroupObject} from "core-app/modules/hal/resources/wp-collection-resource";
+import {SchemaCacheService} from "core-components/schemas/schema-cache.service";
+import {
+  groupIdFromIdentifier,
+  groupTypeFromIdentifier
+} from "core-components/wp-fast-table/builders/modes/grouped/grouped-rows-helpers";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/68fe404308779a812b835bd88ea87fb04c53460e/frontend/src/app/components/wp-table/timeline/container/wp-timeline-container.directive.ts#L69"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>